esrally/metrics.py

```diff
@@ -114,6 +114,27 @@ def guarded(self, target, *args, **kwargs):
 
             try:
                 return target(*args, **kwargs)
+            except elasticsearch.helpers.BulkIndexError as e:
```

Comment:
Can you please move this block just before `except ApiError` to highlight that they are similar?

Reply:
Addressed in https://github.com/elastic/rally/pull/1703/commits/1d88020e21109b6afd1ddb2295e4c05ea8dd78fd